src/global/globalNode.js

```diff
@@ -19,14 +19,15 @@ import Node from '../base/node';
 import FocusedNodeView from '../focused/focusedNodeView';
 
 class GlobalNode extends Node {
-  constructor (node) {
+  constructor (node, entryNode) {
     node.size = node.size || 120;
-    super(node, 'global');
+    super(node, 'global', entryNode);
     this.refreshLoaded();
   }
 
   isInteractive () {
-    return !this.isEntryNode();
+    return true;
```

Comment:
We don't want our entry node to be interactive, so this breaks our use case. I really like your idea of somehow coordinating with if there aren't child nodes. Could we do that here?